app/controllers/annotations_controller.rb

```diff
@@ -352,6 +359,10 @@ def destroy
       p 'did not find @annotation for destroy: ' + params['id']
       format.json { render json: nil, status: :ok }
     else
+      puts @annotation.inspect
```

Comment:
We should work to remove any `puts` or `p` lines from the code. It makes test execution really ugly.